docs/data/evidence-logs.md

```diff
@@ -78,7 +78,7 @@ The Percolator was instructed to visit a URL to see if it is an article landing
 
 ### Percolator attempted to match observation
 
-The Percolator attempted to match one of a number of types of Observation to a DOI. The log entry records the type of observation, the success or failure of the operation, and the resulting DOI.
+The Percolator attempted to match one of a number of types of observation to a DOI. The log entry records the type of observation, the success or failure of the operation, and the resulting DOI.
```

Comment:
'Observation' is a specific thing in the system. I would argue for capitalization case again.